src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is implemented to handle cases where certain properties might be missing, such as `age` in the `olya` and `alex` objects. Consider providing default values or handling undefined values within the `Person` component.